tooling/cli.js/bin/tauri.js

```diff
@@ -74,10 +75,21 @@ ${chalk.yellow('Options')}
     if (process.argv && process.env.NODE_ENV !== 'test') {
       process.argv.splice(0, 3)
     }
+    let cwd = null
+    if (existsSync('package.json')) {
```

Comment:
we could enhance this a bit using https://github.com/sindresorhus/pkg-up , I know I sometimes run npm clis in a nested directory

Reply:
good idea, gonna do it tomorrow

Reply:
pkg-up is literally just calling `findUp('package.json', {cwd})` from https://github.com/sindresorhus/find-up, so I'd say use `find-up` directly

Reply:
done